templates/aem-tools/content-healthcheck.py.epp

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
Just adding a note that we still have to improve consistency across various python code (logging mechanism, args parsing, etc). That will be done as part of https://github.com/shinesolutions/aem-aws-stack-provisioner/issues/6 .